components/StandardButton.qml

```diff
@@ -102,7 +99,14 @@ Item {
                 when: !button.enabled
                 PropertyChanges {
                     target: buttonRect
-                    color: MoneroComponents.Style.buttonBackgroundColorDisabled
+                    opacity: 0.5
```

Comment:
Would the opacity automatically go to 1 again if it jumps from `disabled` to `active`? It seems like it never gets reset.

Reply:
Yes, when going from disabled to active it resets to the default value (opacity = 1).